dask_jobqueue/tests/test_pbs.py

```diff
@@ -34,7 +34,7 @@ def test_header(Cluster):
 
         assert '#PBS -j oe' not in cluster.job_header
         assert '#PBS -N' in cluster.job_header
-        assert '#PBS -l select=1:ncpus=' in cluster.job_header
+        # assert '#PBS -l select=1:ncpus=' in cluster.job_header
```

Comment:
If we fail fast if enough information not provided, and if we give a default cores number, this assert should be ok.